_includes/topnav.html

```diff
@@ -51,21 +50,21 @@
                 {% endfor %}
                 {% if site.feedback_disable == null or site.feedback_disable == false %}
 			{% include feedback.html %}
-    {% endif %}
-                <li>
-                    <form name="search" class="input-group lb-search" action="/search" method="GET">
-                        <input type="text" class="form-control search-bar" placeholder="Search (powered by Watson)" name="q">
-                        {% if page.sidebar contains "lb" %}
-                          <input type="hidden" name="sidebar" value="{{page.sidebar}}">
-                        {% endif %}
-                        <div class="input-group-btn">
-                            <button type="submit" class="btn btn-default" aria-label="search">
-                              <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
-                            </button>
-                        </div>
-                      </form>
-                </li>
-            </ul>
-        </div>
-      </div>
-</nav>
+      {% endif %}
+      <li class="nav-item">
+        <form name="search" class="input-group" action="/search" method="GET">
+            <input type="text" class="form-control" placeholder="Search (powered by Watson)" name="q">
+            {% if page.sidebar contains "lb" %}
+              <input type="hidden" name="sidebar" value="{{page.sidebar}}">
+            {% endif %}
+            <div class="input-group-append">
+                <button type="submit" class="btn btn-secondary" aria-label="search">
+                  <span class="fa fa-search" aria-hidden="true"></span>
+                </button>
+            </div>
+          </form>
+        </li>
+      </ul>
+    </nav>
+  </header>
+</div>
```

Comment:
need a new line here

Reply:
Try it if you need it and i'll fix the problem.

Reply:
I don't think it makes a difference with or without the new line. However, the linter is complaining about it.